.github/bors.toml

```diff
@@ -5,9 +5,17 @@
 #
 # This is left at being compile only as a rough heuristic to detect merge-skew.
 # We don't bother running/waiting on the full CI for merge commits.
-status = ["Compile Builds (Unit Tests)"]
+#
+# TODO(irfansharif): It'd be nice if we could use the build/builder.sh checkout
```

Comment:
This _may_ be possible, but I think it's difficult to achieve. 

In the current setup, once a cockroach binary is built on a given agent and the caches are warmed, future builds on that same agent will run faster as they use the cached files where possible. There is a random element of being lucky to get on an already primed agent.

Achieving this would require something like one of:

- revamping caching so artifacts are cached outside the agent servers (bazel?)
- building all the artifacts into a docker image and pushing that somewhere (vs. running the docker image and mounting volumes to local directory locations for where the artifacts are deposited) and then somehow informing bors about which image to pull and use (and which one to pull when its merging > 1 PR at once)